libs/providers/flagd-rest/README.md

```diff
@@ -0,0 +1,17 @@
+# NodeJS Flagd REST Provider for OpenFeature
+
+![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
+
+## Installation
+
+```
+$ npm install @openfeature/flagd-rest-provider
+```
+
+## Building
+
+Run `nx package providers-flagd-rest` to build the library.
```

Comment:
We should probably either wrap this in a package json script, or add `npx` to the command. I dont have `nx` install globally on purpose to use project local vertsions.